eclipse/sw360
# feat(ui): Send to fossology error message.

* Added a dialog with message "Fossology is not configured" instead of the confusing message "Error" on clicking of send to fossology icon.
 * Added a switch in sw360.properties to enable or disable the the fossology connection.
 * Displayed the enabled/disabled  status in fossology admin portlet.

**Steps to Test:**

- Go to Projects-> Click on the "Send to Fossology" icon.
- Go to Components -> Release Overview -> Click on the "Send to Fossology" icon.
- Go to Projects -> Clearing Status -> Click on the "Send to Fossology" icon.
- Go to Components -> Clearing Details -> Click on the "Send to Fossology" icon.
In all the above cases, if the fossology connection is disabled, the user gets a dialog message that fossology is not configured.

Go to Admin-> Fossology -> A additional message should appear displaying the fossology connection status(i.e enabled/disabled).

Closes #494 

Signed-off-by: Smruti Sahoo <smruti.sahoo@siemens.com>
